### ISSUE TITLE
Old-style middleware (deprecation warning from aiohttp)

### ISSUE THREAD

**Issue by gyermolenko** [hzlmn/aiohttp-jwt#126]
See https://github.com/aio-libs/aiohttp/pull/2252
where old-style (via factories) was changed to decorators 

```
.../aiohttp/web_app.py:227: DeprecationWarning: old-style middleware "<function JWTMiddleware.<locals>.factory at 0x109b959e0>" deprecated, see #2252
    self._middlewares_handlers = tuple(self._prepare_middleware())
```

**Comment by gyermolenko** [comment]
oh, I'm sorry
I've found merged https://github.com/hzlmn/aiohttp-jwt/commit/ba79237a2cda6096f8b11e9fc6b8248193dbb09b but it is not yet released AFAIC

**Comment by hzlmn** [comment]
Yep, my bad :( I will release all updates in few hours.

**Comment by hzlmn** [comment]
Released `0.5.0`
